Tests/MSFT_xDhcpServerAuthorization.Tests.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+
+if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
+{
+    $PSScriptRoot = [System.IO.Path]::GetDirectoryName($MyInvocation.MyCommand.Path)
+}
+
+$ErrorActionPreference = 'Stop'
```

Comment:
This is the more strict errorActionPref.  So, I would suggest testing with this.